### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Bifabig**
Completed the popup window implementation using dynamic `JavaScript` and DOM manipulation.

- Project section cards are all generated dynamically.
- Popup window generated dynamically.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -6,13 +6,120 @@ const menuIcon = document.querySelector('.menuIcon');
 const menuItem = document.querySelector('.menu-item-desktop');
 const items = document.querySelectorAll('.menuItemDesktop');
 
+const body = document.querySelector('body');
+const cardData = [
+  {
+    name: 'Project 1',
+    description:
+      'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ipsam nobis nemo vitae tempora et magni in corporis earum illum cupiditate repudiandae, amet nihil reprehenderit voluptatum tempore eligendi deserunt id adipisci.',
```
Comment (nahnahdev) [inline script.js]: - [x] Great job so far team :100: I will suggest that you increase the font size of your popup paragraph text. Currently, it looks smaller than in the Figma design :+1: 
- Your Design
image

- The Figma design

| Figma Design | Styles |
|--------|--------|
| image | image |

File: script.js [hunk 2]
```diff
@@ -6,13 +6,120 @@ const menuIcon = document.querySelector('.menuIcon');
...
+    technologies: ['HTML/CSS', 'Ruby on Rails', 'JavaScript'],
+    linklive: 'http://google.com',
+    linksrc: 'http://github.com',
+  },
+  {
+    name: 'Project 2',
+    description:
+      'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ipsam nobis nemo vitae tempora et magni in corporis earum illum cupiditate repudiandae, amet nihil reprehenderit voluptatum tempore eligendi deserunt id adipisci.',
+    featuredimg: './assets/img/laptop 2.png',
+    technologies: ['HTML/CSS', 'Ruby on Rails', 'JavaScript'],
```
Comment (nahnahdev) [inline script.js:25]: - [x] Lastly, please style the middle button so it looks similar to the other two :+1: 
- Your design
image

- The Figma design
image

File: script.js [hunk 3]
```diff
@@ -6,13 +6,120 @@ const menuIcon = document.querySelector('.menuIcon');
...
+const body = document.querySelector('body');
+const cardData = [
+  {
+    name: 'Project 1',
+    description:
+      'Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ipsam nobis nemo vitae tempora et magni in corporis earum illum cupiditate repudiandae, amet nihil reprehenderit voluptatum tempore eligendi deserunt id adipisci.',
+    featuredimg: './assets/img/laptop.png',
+    technologies: ['HTML/CSS', 'Ruby on Rails', 'JavaScript'],
+    linklive: 'http://google.com',
+    linksrc: 'http://github.com',
```
Comment (nahnahdev) [inline script.js:18]: - [x] Lastly, I noticed that whenever I hover over the two buttons, the text is no longer visible. I have attached images below. I will suggest that you refer to the Figma design and apply the right hover interaction styles to the buttons depending on the colors :100: 
- [x] KIndly do same for all the popup buttons :+1: 
- Your Design on hover
image

- The Figma Design
image

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @Bifabig and @Theodoraldo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Popup works as expected :heavy_check_mark:
- No linter errors :heavy_check_mark:
- Professional Readme :heavy_check_mark: 
- Project section generated dynamically from javascript :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @Bifabig        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] linters are passing
- [x] Professional README 💯 
- [x] Nice UI design 🥇 
 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Pull request merged**